include/COpenGLStateManager.h

```diff
@@ -1,22 +1,22 @@
-// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
-// This file is part of the "Nabla Engine".
-// For conditions of distribution and use, see copyright notice in nabla.h
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
 

```

Comment:
@Crisspl do we even use this file anymore?

Reply:
hm it is included in COpenGLExtensionHandler.h. It contains some platform-dependent `#include`s but other than that just dead stuff. I guess it'll be completely useless once opengl function table (or whatever it was called) gets merged (or was it already?)

Reply:
ah  it interferes with @Hazardu work on runtime OGL loading.

Reply:
dont touch right now, then.